tools/osx/crosstool/cc_toolchain_config.bzl

```diff
@@ -5194,20 +5194,12 @@ def _impl(ctx):
     else:
         unfiltered_compile_flags_feature = None
 
+    do_not_split_linking_cmdline = feature(name = "do_not_split_linking_cmdline")
     linker_param_file_feature = feature(
         name = "linker_param_file",
         flag_sets = [
             flag_set(
-                actions = all_link_actions,
-                flag_groups = [
-                    flag_group(
-                        flags = ["-Wl,@%{linker_param_file}"],
-                        expand_if_available = "linker_param_file",
-                    ),
-                ],
-            ),
-            flag_set(
-                actions = [ACTION_NAMES.cpp_link_static_library],
+                actions = all_link_actions + [ACTION_NAMES.cpp_link_static_library],
```

Comment:
We might want to expand this to the objc linking actions as well? (`all_link_actions` does not contain those)